src/services/MeasureService.ts

```diff
@@ -12,10 +15,16 @@ export class MeasureService implements Service<fhir4.Measure> {
     throw new Error('Not implemented');
   }
 
-  // TODO: Remove ts-ignore comment when actually implementing this
-  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-  // @ts-ignore
-  async searchById() {
-    throw new Error('Not implemented');
+  /**
+   *
+   * result of sending a GET request to {BASE_URL}/4_0_1/Measure/{id}
+   * searches for the measure with the passed in id
+   */
+  async searchById(args: RequestArgs) {
+    const result = await findResourceById(args.id, 'Measure');
```

Comment:
Thoughts on logging? We should probably log something about when each service function is called, maybe something like:

```ts
logger.info(`GET /Measure/${args.id}`);
```

requires importing the logger in this file. See `index.ts` for an example of that

Reply:
^ I agree - I think this would be a good place to do some logging, and Matt's example seems good since it's straight to the point